api/v1/errors/SqlError.js

```diff
@@ -0,0 +1,28 @@
+function checkError(err) {
+        if (!err) return false;
+        var re = /ER_DUP_ENTRY|ER_NO_REFERENCED_ROW/;
+        return re.test(err.message);
+};
+
+
+var UnprocessableRequestError = require('./UnprocessableRequestError.js');
```

Comment:
This should probably be derived from an API error tbh. If there's a SQL error, that's our fault.